config/defaults_gen/defaults_gen.go

```diff
@@ -0,0 +1,179 @@
+// Copyright (C) 2019-2020 Algorand, Inc.
```

Comment:
the name looks confusing, I thought this file is auto-generated

Reply:
do you think that defaults_generator or defaultsGenerator as the package name would be more appropriate ?

Reply:
`defaultsGenerator` seems like a better name